zetaclient/chains/evm/observer/inbound.go

```diff
@@ -794,6 +794,12 @@ func (ob *Observer) ObserveTSSReceiveInBlock(ctx context.Context, blockNumber ui
 				)
 			}
 		}
+		if ethcommon.HexToAddress(tx.From) == ob.TSS().EVMAddress() {
+			nonce := uint64(tx.Nonce)
```

Comment:
will need `// #nosec G115 always positive` to suppress nosec warning

Reply:
are they suppressed already in the CI gosec? I don't see complaints in CI.

Reply:
Not quite sure, maybe the dev ops team suppressed at some point. I see lots of `// $nosec G115` comments out there.